torch/csrc/jit/codegen/cuda/index_compute.cpp

```diff
@@ -1433,9 +1449,52 @@ std::vector<Val*> Index::getGlobalProducerStridedIndices(
     }
   }
 
+  // Save indexing info necessary for validating vectorization at launch time
+  fillProducerVectorizedContigRootDomains(
+      producer_tv, consumer_tv, c2p_map, contig_finder);
+
   return strided_inds;
 }
 
+namespace {
+
+// Maps all producer domains to consumer with broadcast
+// forwarding. Used to find the allocation position.
+std::unordered_map<IterDomain*, IterDomain*> mapAllProducerDomainsToConsumer(
```

Comment:
Moved this to its own function. I incorrectly used `replay_PasC` created here to find mapped domains that should really be done using the CA index map. Extracted that part of the code so as to avoid the same error.